server/api/submission/submission.integration.js

```diff
@@ -2,10 +2,68 @@
 
 var app = require('../..');
 var request = require('supertest');
-
+var User = require('../user/user.model')
```

Comment:
Semicolons are never required.  We (probably) should choose a style and then actually enforce linting, but meh.  http://stackoverflow.com/questions/444080/do-you-recommend-using-semicolons-after-every-statement-in-javascript